### DISCUSSION TITLE
Search Feature

### DISCUSSION THREAD

**Discussion by dhairyasenjaliya** [flyerhq/react-native-chat-ui#62] [Ideas / Feature Requests]
Hello thank you for awesome library 

- so my suggestion is to have search bar at top corner which makes chat-ui complete

**Comment by demchenkoalex** [comment]
Hey! As you can see long time no updates, currently focused on a Flutter chat UI, waiting for either more time or some contributors to help :) As for the search - I don't think it will ever be added tbh, it was never planned, because chat UI is backend agnostic and for a normal use case with pagination, search must be a part of a backend part, not chat UI. As for the search bar you can add it on top yourself, and then write a logic for whatever service you're using.
